packages/pie-models/src/pie/ruler/index.ts

```diff
@@ -0,0 +1,38 @@
+import {PieModel} from '../../PieModel';
+import { PromptConfig } from '../../PromptConfig';
+import { CommonConfigSettings } from '../../CommonConfigSettings';
+
+type RulerPieMetric = 'mm' | 'cm' | 'm' | 'km';
```

Comment:
why not just `Metric` and `Imperial`?

Reply:
For no reason, thank you for the suggestion. I will update it.